Kernel/Arch/aarch64/RegisterState.h

```diff
@@ -23,6 +23,7 @@ struct alignas(16) RegisterState {
     u64 elr_el1;   // Exception Link Register, EL1
     u64 sp_el0;    // EL0 stack pointer
     u64 tpidr_el0; // EL0 Software Thread ID Register
+    u64 esr_el1;
```

Comment:
rest of these have a comment describing what the register means

Reply:
Oh I noticed that as well just now